code/modules/mob/living/silicon/robot/component.dm

```diff
@@ -98,25 +96,21 @@
 	name = "radio"
 	external_type = /obj/item/robot_parts/robot_component/radio
 	energy_consumption = 1
-	max_damage = 40
 
 /datum/robot_component/binary_communication
 	name = "binary communication device"
 	external_type = /obj/item/robot_parts/robot_component/binary_communication_device
 	energy_consumption = 0
-	max_damage = 30
 
 /datum/robot_component/camera
 	name = "camera"
 	external_type = /obj/item/robot_parts/robot_component/camera
 	energy_consumption = 1
-	max_damage = 40
```

Comment:
Unatomic change, made camera's health drop from 40 to 15

Reply:
intentional